recipes/giflib/5.2.x/CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.1...3.31.0)
```

```suggestion
cmake_minimum_required(VERSION 3.15)
```
Use this one instead. The 3.1 CMake policy version is simply too old for this CMakeLists.txt. `IN_LIST` is only available since 3.3.

CMake version 3.15 is the mandatory minimum since Conan v2 anyway, so it's a good one to use.